lib/llm/OpenAIClient.ts

```diff
@@ -138,10 +147,47 @@ export class OpenAIClient extends LLMClient {
       },
     });
 
-    const response = await this.client.chat.completions.create({
+    delete openAiOptions.requestId;
+    delete openAiOptions.image;
```

Comment:
would strongly prefer not to have `delete`; instead even if tedious, let's make a new variable with all the same fields sans `requestId` and `image`